enterprise/frontend/src/embedding-sdk/components/private/SdkLicenseProblemBanner/SdkLicenseProblemBanner.module.css

```diff
@@ -0,0 +1,34 @@
+.PopoverDropdown.PopoverDropdown {
+  border: none;
+}
+
+.Indicator {
+  cursor: pointer;
+  border-radius: 4px;
+  box-shadow: 0 6px 24px 0 #0000001a;
+}
+
+.Indicator.Warning {
+  border: 1px solid #e8b923;
+}
+
+.Indicator.Error {
+  border: 1px solid #f7a4a3;
+}
```

Comment:
These colors doesn't have to be themed as it's not user-facing, and it's not the same as our CSS variables - the color differs a bit. I'll try to see if I can use `color-mix(...)` to derive these colors from our base palette.